### ISSUE TITLE
[Push Publishing] : Single quote in content's title breaks JavaScript code in the portlet

### ISSUE THREAD

**Issue by jcastro-dotcms** [dotCMS/core#21699]
Reported via Support ticket: [106480](https://dotcms.zendesk.com/agent/tickets/106480)
Reproducible as of dotCMS 5.3.8.9, up to latest master (22.01)

A JavaScript error is being generated when the value of the `Title` field of an HTML Page or Contentlet has at least a single quote in it.

**To Reproduce**
1. Start up a single instance and set up Push Publishing -- **you DO NOT need to set a receiving endpoint**.
2. Create a test HTML Page and use a single quote in its title: `ABC follows MyBank's lead, this is some additional copy`
3. Push Publish this new page, and go to the **Site > Publishing Queue** portlet.
4. You'll get JavaScript errors in the browse's console when you click the box to select all Bundles, to view their details, or when you click the red `DELETE` button. In Firefox, you'll see this:
```
Uncaught SyntaxError: missing } after property list
    Dojo 40
    Angular 2
    Dojo 64
[layout:174:24](https://demo.dotcms.com/c/portal/layout?p_l_id=b7ab5d3c-5ee0-4195-a17e-8f5579d718dd&p_p_id=publishing-queue&p_p_action=0&&dm_rlout=1&r=1644530094146&in_frame=true&frame=detailFrame&container=true&angularCurrentPortlet=publishing-queue)note: { opened at line 173, column 11[layout:173:11](https://demo.dotcms.com/c/portal/layout?p_l_id=b7ab5d3c-5ee0-4195-a17e-8f5579d718dd&p_p_id=publishing-queue&p_p_action=0&&dm_rlout=1&r=1644530094146&in_frame=true&frame=detailFrame&container=true&angularCurrentPortlet=publishing-queue)
```
**NOTE TO DEVS: The browser may sometimes cache the behavior and allow you to select the Bundle and even view its details. However, (1) it will not show all the items that you added to the bundle (if you used the `Add To Bundle` feature). Once you refresh the browser's tab, it will show you the original problem.**

**Expected behavior**
The system must escape any potentially harmful characters present in any user input.

**Additional context**
Considering this a very specific code issue, the only workaround is clearing the Push Publishing History via SQL queries.

**Comment by jcastro-dotcms** [comment]
Another customer report: https://dotcms.zendesk.com/agent/tickets/106535

**Comment by dsilvam** [comment]
Passed Internal QA: Followed steps in the description and not getting errors.

**Comment by bryanboza** [comment]
Fixed, tested on release-22.03 // Docker // MSSQL. I'm unable to reproduce the described behavior after the changes.
